<launch>
  <node name="ahrs" pkg="engy" type="ahrs_read"/>
  <node name="analog_input" pkg="engy" type="volt_meter_12bit.py"/>
  <node name="rc_direct" pkg="engy" type="rc_direct_control.py"/>
  <node name="esc_all" pkg="engy" type="esc_all.py"/>
  <node name="esc_surge" pkg="engy" type="esc_surge.py"/>
  <node name="led_light_set" pkg="engy" type="led_light.py"/>
  <node name="camera_pantilt" pkg="engy" type="camera_pantilt.py"/>

  <node pkg="uvc_camera" name="camera1" type="uvc_camera_node" output="screen">
    <param name="device" type="string" value="/dev/video0"/>
    <param name="width" type="int" value="848"/>
    <param name="height" type="int" value="480"/> <!-- 960 540 800 600-->
    <param name="pixel_format" value="mjpeg"/>
    <param name="camera_frame_id" value="usb_cam"/>
    <param name="io_method" value="mmap"/>
    <param name="frame_id" value="camera"/>
    <!--param name="camera_info_url" type="string" value="file:///home/{user_name}/.../camera.yaml"/-->
  </node>

  <node name="web_video_server" pkg="web_video_server" type="web_video_server" />
  <node name="depth_node" pkg="engy" type="depth_node.py"/>
  
  <!--node name="camera_focus_set" pkg="engy" type="camera_focus.py"/-->



</launch>